File: program/program/convert-inches-to-feet/ConvertInchesToFeet.cs
using System;

class MainClass {
    public static void Main (string[] args) {
    Console.WriteLine ("Please enter inches:");
    double inches = Convert.ToDouble(Console.ReadLine());
    double feet = inches / 12;
    Console.WriteLine(inches+ " inches is "+feet + " Feet");
  }
}

File: program/program/convert-inches-to-feet/convert_inches_to_feet.rs
fn main() {
    // Prompt the user for the number of inches
    println!("Enter the number of inches: ");

    // Read the user input and store it in a variable
    let mut inches = String::new();
    std::io::stdin().read_line(&mut inches)
        .expect("Failed to read line");

    // Check if the input is empty
    if inches.trim().is_empty() {
        println!("Please enter a valid number");
        return;
    }

    // Convert the input to a number
    let inches: u32 = inches.trim().parse()
        .expect("Please enter a valid number");

    // Convert inches to feet
    let feet = inches / 12;

    // Print the result
    println!("{} inches is equal to {} feet", inches, feet);
}
